blog/_posts/2024-02-26-scala-3.4.0-and-3.3.2-released.md

```diff
@@ -0,0 +1,233 @@
+
+## What's new in 3.3.2 LTS (and 3.4.0 too)
+
+In the release notes of Scala [3.3.2 LTS](https://github.com/lampepfl/dotty/releases/tag/3.3.2), you can see a lot of bug fixes. One area that received special attention in that regard was coverage support. With most pains fixed, we are now confident in the state of coverage.
+
+Another important change, not directly visible to end users, is the integration of the presentation compiler into the compiler itself. This makes building tooling easier and allows for a more stable and reliable user experience when using Metals.
+
+## Changes exclusive to 3.4.0
+
+Release notes of [3.4.0](https://github.com/lampepfl/dotty/releases/tag/3.4.0) contains many substantial changes and improvements. The most noteworthy can be grouped in a few broad categories:
+
+### Stabilized SIPs
+
+- [SIP-54](https://docs.scala-lang.org/sips/multi-source-extension-overloads.html) is now a standard feature. It allows for importing extension methods with the same name from different sources.
+- [SIP-53](https://docs.scala-lang.org/sips/quote-pattern-type-variable-syntax.html) is now a standard feature. It allows for more expressive type patterns in quotes. For example:
+
+  ```scala
+  case '[ (t, t, t) ] => ???
+  ```
+
+  will match any 3-element tuple, and `t` will be the greatest lower bound of types of all three elements.
+- Match types are now properly specified, and thanks to that, they work in a more predictable and stable way. See [SIP-56](https://docs.scala-lang.org/sips/match-types-spec.html)
+
+### Improvements to the type system and inference
+
+- Type inference for functions similar to fold is greatly improved. For example:
+
+  ```scala
+  def partition[T](xs: List[T], pred: T => Boolean) =
+     xs.foldRight((Nil, Nil)): (x, p) =>
+       if pred(x) then (x :: p._1, p._2) else (p._1, x :: p._2)
+
+  ```
+
+  will have its return type correctly inferred as `(List[T], List[T])`. Earlier, it would result in a rather unintuitive type error.
+
+- It is now possible to write a polymorphic lambda without writing down the types of its value parameters as long as they can be inferred from the context, for example:
+
+  ```scala
+  val f: [T] => T => String = [T] => (x: T) => x.toString
+  ```
+
+  can now be written more concisely as:
+
+  ```scala
+  val f: [T] => T => String = [T] => x => x.toString
+  ```
+
+- The compiler now avoids generating given definitions that loop, removing a long-standing footgun in implicit resolution.
+
+### Backend improvements
+
+- Polymorphic lambdas are now implemented using JVM lambdas when possible instead of anonymous classes, which will make them more efficient.
+- JVM Backend parallelization has been ported from Scala 2 to Scala 3. Read more in the discussion under [the original PR](https://github.com/scala/scala/pull/6124).
+
+### Reporting
```

Comment:
What about a mention of https://github.com/lampepfl/dotty/pull/17835, which exposes unused warnings to semanticdb-based tooling? I haven't seen it in the RCs release notes either, I don't know if that's expected?

Reply:
Right we probably overlooked adding the "release-notes" label